Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Issue.hs

```diff
@@ -186,13 +204,67 @@ createIssueReport (driverId, _) Common.IssueReportReq {..} = do
             categoryId = cast categoryId,
             mediaFiles = cast <$> mediaFiles,
             assignee = Nothing,
-            status = D.NEW,
+            status = D.OPEN,
             deleted = False,
+            ticketId = Nothing,
             createdAt = now,
             updatedAt = now,
             description
           }
 
+    mkTicket :: D.IssueReport -> D.IssueCategory -> Maybe D.IssueOption -> Maybe Ride.Ride -> ShortId DM.Merchant -> [Text] -> Flow Kapture.CreateTicketReq
+    mkTicket issue category mbOption mbRide shortId mediaFileUrls = do
+      info <- mkRideInfo mbRide shortId
+      person <- QP.findById driverId >>= fromMaybeM (PersonNotFound driverId.getId)
+      return $
+        Kapture.CreateTicketReq
+          { category = category.category,
+            subCategory = (.option) <$> mbOption,
+            issueId = Just issue.id.getId,
+            issueDescription = description,
+            mediaFiles = Just mediaFileUrls,
+            name = Just $ person.firstName <> " " <> fromMaybe "" person.lastName,
+            phoneNo = person.unencryptedMobileNumber,
```

Comment:
shouldn't we `decrypt person.mobileNumber` instead of using `unencryptedMobileNumber` @hkmangla ?